ga4gh/server/cli/server.py

```diff
@@ -52,7 +93,10 @@ def server_main(args=None):
     sslContext = None
     if parsedArgs.tls or ("OIDC_PROVIDER" in frontend.app.config):
         sslContext = "adhoc"
-    frontend.app.run(
-        host=parsedArgs.host, port=parsedArgs.port,
-        use_reloader=not parsedArgs.dont_use_reloader,
-        ssl_context=sslContext)
+    if parsedArgs.debug:
```

Comment:
I don't get why this is called `debug` ... seems like this should be `run_guincorn` or something...